org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/jobs/AbstractAnalyzeProjectJob.java

```diff
@@ -78,20 +79,30 @@ public abstract class AbstractAnalyzeProjectJob<CONFIG extends AbstractAnalysisC
   private final TriggerType triggerType;
   private final boolean shouldClearReport;
   private final Collection<FileWithDocument> files;
-
+  private EnumSet<Language> unavailableLanguagesReference;
```

Comment:
Even though Java does pass by values, this is actually a reference. There is no way to denote it this way in Java, that's why I named it this way.

Reply:
This was removed